File: BasicAuthentication/ViewModels/Requests/Login/RequestLogin.cs
using System.ComponentModel.DataAnnotations;

namespace BasicAuthentication.ViewModels.Requests.Login;

/// <summary>
/// 로그인 요청 클래스
/// </summary>
public class RequestLogin
{
    /// <summary>
    /// 로그인 아이디
    /// </summary>
    [Required(ErrorMessage = "로그인 아이디를 입력해주세요")]
    public string LoginId { get; set; }

    /// <summary>
    /// 패스워드
    /// </summary>
    [Required(ErrorMessage = "패스워드를 입력해주세요")]
    public string Password { get; set; }
}

File: BudgetBringerV2/src/models/requests/login/request-login.ts
/**
 * Request Login Model
 */
export class RequestLogin {
  // Id
  loginId : string = '';

  // Password
  password : string = '';

  /**
   * Validator
   */
  isInvalid() {
    return this.loginId === '' || this.password === '';
  }
}
